lit_gpt/config.py

```diff
@@ -40,15 +43,29 @@ class Config:
     # credit https://arxiv.org/pdf/2305.13245.pdf
     n_query_groups: Optional[int] = None
     shared_attention_norm: bool = False
+    _norm_class: str = "LayerNorm"
```

Comment:
This underscore quirk is because we cannot have these be the modules themselves or the dataclass won't be serializable. See the added properties below.